contrib/python-bindings/tests/mapping_wrapper.py

```diff
@@ -0,0 +1,37 @@
+# ---------------------------------------------------------------------
+#
+# Copyright (C) 2016 by the deal.II authors
+#
+# This file is part of the deal.II library.
+#
+# The deal.II library is free software; you can use it, redistribute
+# it, and/or modify it under the terms of the GNU Lesser General
+# Public License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+# The full text of the license can be found in the file LICENSE.md at
+# the top level directory of deal.II.
+#
+# ---------------------------------------------------------------------
+
+import unittest
+from PyDealII.Debug import *
+
+class TestMappingWrapper(unittest.TestCase):
+
+    def setUp(self):
+        self.triangulation = Triangulation('2D')
+        self.triangulation.generate_hyper_cube()
+        self.triangulation.refine_global(1)
+
+    def test_mapping(self):
+        mapping = MappingQGeneric(2,2,1)
```

Comment:
Can you add a test with `spacedim > dim`, just to make sure that we got all the `dim` and `spacedim` correct.